activerecord/lib/active_record/tasks/database_tasks.rb

```diff
@@ -241,6 +241,23 @@ def migrate
         Migration.verbose = verbose_was
       end
 
+      def migrate_all
```

Comment:
@jonathanhefner It turns out adding a `migrate_all` to `DatabaseTasks` was pretty straight forward.